README.md

```diff
@@ -1,13 +1,13 @@
 # Vorpal
 
-
 [![Build Status](https://travis-ci.org/dthree/vorpal.svg)](https://travis-ci.org/dthree/vorpal/)
 <a href="https://www.npmjs.com/package/vorpal">
-  <img src="https://img.shields.io/npm/dt/vorpal.svg" alt="NPM Downloads" />
+<img src="https://img.shields.io/npm/dt/vorpal.svg" alt="NPM Downloads" />

```

Comment:
Unrelated to the PR, but at some point we should convert that to simple markdown :)